docs/user-guide/deployments.md

```diff
@@ -76,47 +76,34 @@ The created Replica Set will ensure that there are three nginx Pods at all times
 
 ## The Status of a Deployment
 
-After creating or updating a Deployment, you would want to confirm whether it succeeded or not. The best way to do this is through checking its status.
-
-To verify if the above Deployment succeeded or not, first compare the `.metadata.generation` and `.status.observedGeneration` of the Deployment:
+After creating or updating a Deployment, you would want to confirm whether it succeeded or not. The simplest way to do this is through `kubectl rollout status`.
 
 ```shell
-$ kubectl get deployment/nginx-deployment -o yaml | grep [Gg]eneration
-  generation: 2
-  observedGeneration: 2
+$ kubectl rollout status deployment/nginx-deployment
+<some more messages if update is in progress>
```

Comment:
I assume this is not actual output? It would be more clear to put that explanation above/below this example. Maybe do it again after `set image`, for instance, to show a rollout in progress.